src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -98,6 +106,16 @@ public static ProviderCultureResult ParseCookieValue(string value)
             var cultureName = potentialCultureName.Substring(_culturePrefix.Length);
             var uiCultureName = potentialUICultureName.Substring(_uiCulturePrefix.Length);
 
+            if (!IsValidCulture(cultureName))
```

Comment:
Should this go after the null check on Line 101 \ 119? Presumably `GetCultureInfo` will throw a arg null if it's null.